tests/console/zypper_ar.pm

```diff
@@ -22,10 +22,11 @@ sub run {
     if (my $urlprefix = get_var('MIRROR_PREFIX')) {
         my @repos_to_add = qw(OSS NON_OSS OSS_DEBUGINFO);
         my $repourl;
+        my $repos = script_output "zypper lr";
```

Comment:
I consider this a quite unsafe approach. Maybe we can find a way to solve it just within the SUT? See further down